### ISSUE TITLE
'com.okta.telephony.provider' not supported for inline hooks

### ISSUE THREAD

**Issue by mcrobbj-SilverBullet** [okta/terraform-provider-okta#1128]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
v1.1.9

### Affected Resource(s)

- okta_inline_hook

### Terraform Configuration Files

resource "okta_inline_hook" "twillio" {
  name    = "twillio"
  version = "1.0.0"
  type    = "com.okta.telephony.provider"

  channel = {
    version = "1.0.0"
    uri     = "https://example.com/test"
    method  = "POST"
  }

  auth = {
    key   = "Authorization"
    type  = "HEADER"
    value = "secret"
  }
}

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Expected Behavior
Add inline hook

### Actual Behavior

│ Error: expected value to be one of 'com.okta.oauth2.tokens.transform', 'com.okta.import.transform', 'com.okta.saml.tokens.transform', 'com.okta.user.pre-registration', 'com.okta.user.credential.password.import', got 'com.okta.telephony.provider'
│ 
│   with okta_inline_hook.twillio,
│   on main.tf line 273, in resource "okta_inline_hook" "twillio":
│  273:   type    = "com.okta.telephony.provider"

It is one of the supported providers (early access)

### Steps to Reproduce

1. `terraform apply`

### References

- #0000

**Comment by monde** [comment]
Thanks @mcrobbj-SilverBullet , I did a quick fix #1132 , it will go out today when I do a v3.28.0 release.

**Comment by monde** [comment]
Released as [v3.28.0](https://github.com/okta/terraform-provider-okta/releases/tag/v3.28.0).
